CombinatorialGames/Surreal/Basic.lean

```diff
@@ -371,5 +371,36 @@ theorem toGame_sub (x y : Surreal) : toGame (x - y) = toGame x - toGame y :=
 theorem toGame_natCast (n : ℕ) : toGame n = n :=
   map_natCast' toGameAddHom rfl n
 
+@[simp]
+theorem game_out_eq (x : Surreal) : Game.mk x.out = x.toGame := by
+  cases x
+  rw [toGame_mk, Game.mk_eq_mk]
+  exact mk_out_equiv _
+
+/-- Construct a `Surreal` from its left and right sets, and a proof that all elements from the left
+set are less than all the elements of the right set.
+
+Note that although this function is well-defined, recovering the left/right sets from a surreal
```

Comment:
Perhaps be more clear about the lack of Surreal representatives for Surreals of infinite birthday?

... is well-defined, this function isn't injective nor do equivalence classes in `Surreal` have a well-defined representative ...

Reply:
I've updated the docstring.